posthog/migrations/0799_alter_survey_type.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 4.2.22 on 2025-07-18 20:08
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("posthog", "0798_alter_integration_kind"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="survey",
+            name="type",
+            field=models.CharField(
+                choices=[
+                    ("popover", "popover"),
+                    ("widget", "widget"),
+                    ("button", "button"),
+                    ("email", "email"),
+                    ("full_screen", "full screen"),
+                    ("external_survey", "external survey"),
```

Comment:
what's the difference between full_screen and external_survey here?

Reply:
so besides the three ones we already have (popover, api, feedback button), we also had button, email and full_screen. We don't use them anywhere, but I preferred to not touch them. do you think se should remove them?